attributioncode/validator.go

```diff
@@ -201,3 +215,11 @@ func checkMAC(key, msg, msgMAC []byte) error {
 	}
 	return nil
 }
+
+func fromRTAMO(content string) bool {
+	matched, err := regexp.MatchString(`^rta:`, content)
```

Comment:
Should the regexp declaration be made static instead of initializing inside this function for every request?

Reply:
Do you mean like compiling the regex first, and then using that object to compare? The docs say this is fine for simple regex matching, but if your concern is around performance considerations I can look at how to do that

https://pkg.go.dev/regexp#MatchString

Reply:
@jbuck updated - thanks for the feedback!